src/pyhf/tensor/tensorflow_backend.py

```diff
@@ -722,3 +722,28 @@ def transpose(self, tensor_in):
         .. versionadded:: 0.7.0
         """
         return tf.transpose(tensor_in)
+
+    def fisher_cov(self, model, pars, data):
+        raise NotImplementedError
```

Comment:
Is this just technically more difficult or there is some other reason this is not also added?

Reply:
have now added, thanks for flagging!